templates/boot-fastboot/generic-fastboot-boot-template.jinja2

```diff
@@ -0,0 +1,85 @@
+{%- extends 'base/kernel-ci-base.jinja2' %}
+{%- block protocols %}
+protocols:
+  lava-lxc:
+    name: lxc-{{ platform }}-art-conv
+    template: debian
+    distribution: debian
+    release: jessie
+    mirror: http://mirror.bytemark.co.uk/debian
+{%- endblock %}
+
+{% block actions %}
+actions:
+{%- block deploy_lxc %}
+
+  - deploy:
+      namespace: tlxc
+      timeout:
+        minutes: 5
+      to: lxc
+      packages:
+    - android-tools-adb
+    - android-tools-fastboot
+    - mkbootimg
+    - skales
```

Comment:
This skales tool probably deserves a comment since it's not obvious for those unfamiliar with qcom boards.   In fact, you should probably just remove skales for now.  it can be added when this support is extended for boards that need different mkbootimg tools.

Reply:
oky